openapi2conv/openapi2_conv.go

```diff
@@ -424,9 +424,33 @@ func ToV3Response(response *openapi2.Response) (*openapi3.ResponseRef, error) {
 	if schemaRef := response.Schema; schemaRef != nil {
 		result.WithJSONSchemaRef(ToV3SchemaRef(schemaRef))
 	}
+	if headers := response.Headers; len(headers) > 0 {
+		result.Headers = ToV3Headers(headers)
+	}
 	return &openapi3.ResponseRef{Value: result}, nil
 }
 
+func ToV3Headers(defs map[string]*openapi2.Header) openapi3.Headers {
+	headers := make(openapi3.Headers)
+	for name, header := range defs {
+		stripNonCustomExtensions(header.Extensions)
+		if ref := header.Ref; ref != "" {
+			headers[name] = &openapi3.HeaderRef{Ref: ToV3Ref(ref)}
+		} else {
+			headers[name] = &openapi3.HeaderRef{Value: &openapi3.Header{
+				Parameter: openapi3.Parameter{
+					ExtensionProps: header.ExtensionProps,
+					Description:    header.Description,
+					Schema: &openapi3.SchemaRef{Value: &openapi3.Schema{
+						Type: header.Type,
```

Comment:
Why only translate type?

Reply:
because the i is only `type` parameter in the v2 struct: https://github.com/getkin/kin-openapi/blob/master/openapi2/openapi2.go#L234-L239

We can change the v2 Header struct to use Parameter as it is done in v3

Reply:
I see. This is missing plenty: https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md#:~:text=If%20in%20is%20any%20value%20other%20than%20%22body%22%3A 

If you don't feel like adding the missing fields in this PR it can always be done in a follow up PR.

Reply:
I will add in this PR, np

Reply:
ok, I adde second commit here with using the Parameter for Header struct